Zoo/Zoo/Zoo.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+namespace Zoo
+{
+    public class Zoo 
+    {
+        private string nameZoo;
+        private string[] listAnimals = new string [5];
```

Comment:
mai bine ai folosi List<Animal> fiindca iti permite sa tot adaugi. cand aduci in schimb ceva in array, capacitatea lui fiind fixa...ar trebui sa faci array.
Also...Lista de animale...trebuie sa fie...nu string-uri ci typuri reale :)